### PR TITLE
New version: Microsoft.Edge version 95.0.1020.44

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
### ARP entries are correct.

Auto-updated by [vedantmgoyal2009/winget-pkgs-automation](https://github.com/vedantmgoyal2009/winget-pkgs-automation)

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/33993)

### CODE REVIEW

File: manifests/m/Microsoft/Edge/95.0.1020.44/Microsoft.Edge.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+- http
+- https
+FileExtensions:
+- html
+- htm
+- url
+- pdf
+- crx
+Installers:
+- Architecture: x86
```
Comment (ItzLevvie) [inline manifests/m/Microsoft/Edge/95.0.1020.44/Microsoft.Edge.installer.yaml:26]:
```suggestion
- Architecture: arm64
```

File: manifests/m/Microsoft/Edge/95.0.1020.44/Microsoft.Edge.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  InstallerUrl: https://msedge.sf.dl.delivery.mp.microsoft.com/filestreamingservice/files/193387bf-a465-43e8-8975-4abfd182c4cd/MicrosoftEdgeEnterpriseARM64.msi
+  InstallerSha256: 50C98D090D7835293C435C86280E0A5FC487DE5BBDB427AFBA09B3D08D8CB4EB
+  ProductCode: '{C2D7FD19-D089-3E88-AAFC-EDC05CA8DF6A}'
+- Architecture: arm64
```
Comment (ItzLevvie) [inline manifests/m/Microsoft/Edge/95.0.1020.44/Microsoft.Edge.installer.yaml:30]:
```suggestion
- Architecture: x64
```

File: manifests/m/Microsoft/Edge/95.0.1020.44/Microsoft.Edge.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  InstallerUrl: https://msedge.sf.dl.delivery.mp.microsoft.com/filestreamingservice/files/95db1c39-9ea5-496e-8f92-edc6d4f8c5a3/MicrosoftEdgeEnterpriseX64.msi
+  InstallerSha256: 7FF47C76BB4F5A077720370682EAA49AB0F124F56E540F8D0594C1F46FD6A41A
+  ProductCode: '{A70AED58-CAB1-3CDB-B701-D5FD264625E3}'
+- Architecture: x64
```
Comment (ItzLevvie) [inline manifests/m/Microsoft/Edge/95.0.1020.44/Microsoft.Edge.installer.yaml:34]:
```suggestion
- Architecture: x86
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by jedieaston — CHANGES_REQUESTED** [review]
The version number for this is wrong; these MSIs install 95.0.1040.40, which we already have.

95.0.1040.44 has only been released for arm64 so far.

image

image

**Comment by Trenly** [comment]
Still have to fix the version numbers, but I updated the installer architectures to match

**Comment by vedantmgoyal2009** [comment]
When a new version of MS Edge will be available, I could remove x64 and x86 installer links and then it could be merged. If it will be merged now, my automation would break.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
> If this will be merged now, my automation would break.

**Pull request closed without merging**